packages/apidom-ls/src/config/openapi/parameter/lint/style--type.ts

```diff
@@ -5,12 +5,12 @@ import { LinterMeta } from '../../../../apidom-language-types.ts';
 import { OpenAPI3 } from '../../target-specs.ts';
```

Comment:
Name of the files according to established conventions of linting rules production should be `style--equals.ts`